README.md

```diff
@@ -24,14 +24,14 @@ git submodule update --init
 The development web server is started with:
 
 ```bash
-make hugo
-hugo server
+make hugo 
+#or
+python gen_config.py
```

Comment:
This looks very confusing. Why not leave the existing instructions alone, and add below it:

_If you don't have `make` available (e.g., on Windows), use this instead_:
```
python gen_config.py
hugo
hugo server
```

Reply:
oh yes! This is actually better I will do the changes asap!

Reply:
Done! @rgommers